src/neuroconv/datainterfaces/behavior/deeplabcut/deeplabcutdatainterface.py

```diff
@@ -133,6 +144,7 @@ def add_to_nwbfile(
         self,
         nwbfile: NWBFile,
         metadata: Optional[dict] = None,
+        container_name: Optional[str] = "PoseEstimation",
```

Comment:
```suggestion
        container_name: str = "PoseEstimation",
```

I think we can drop the `None` case here to be explicit about the value of the default string